### PR TITLE
fix: digest inifiles correctly

### LINKED ISSUE #62: [TS-634] [BUG] `.bandit` file is used as config, should be ini
## observed 

If `.bandit` file is found, it is run with `bandit -c .bandit`.  (see above)
This is wrong.

## expected
`.bandit` file is expected to be in INI format and must be run with `bandit --ini .bandit`.

https://bandit.readthedocs.io/en/latest/config.html#
>  To use this, put an INI file named `.bandit` in your project’s directory. [...]

## contact

https://github.com/codacy/codacy-bandit/blob/ab42badc52e207130748e968387f5246c09b5ca9/src/main/scala/codacy/bandit/Bandit.scala#L35

BTW: the `bandit.yml` is a config, not an INI, and is correctly called with `bandit -c bandit.yml`

### PR DESCRIPTION
**Pull request by jkowalleck**
fixes #62

### CODE REVIEW

File: src/main/scala/codacy/bandit/Bandit.scala [hunk 1]
```diff
@@ -32,14 +32,21 @@ object Bandit extends Tool {
     }
   }
 
-  private lazy val nativeConfigFileNames = Set("bandit.yml", ".bandit")
+  // https://bandit.readthedocs.io/en/latest/config.html
+  private lazy val nativeIniFileNames = Set(".bandit", "bandit.ini")
+  private lazy val nativeConfigFileNames = Set("bandit.yml", "bandit.yaml", "bandit.toml")
```
[the remark this answers is not in the document]
Comment (jkowalleck) [reply src/main/scala/codacy/bandit/Bandit.scala:37]: code style was not adhered before, see old var names like `nativeConfigFileNames` 
...

**Comment by DMarinhoCodacy** [comment]
created new PR with the same changes. Closing this one

**Comment by jkowalleck** [comment]
what happened, @DMarinhoCodacy ???

my issue #62 was deleted, and the statement from https://github.com/codacy/codacy-bandit/pull/63#issuecomment-2265817409 is just not true.

**Comment by DMarinhoCodacy** [comment]
created this PR https://github.com/codacy/codacy-bandit/pull/73

**Comment by jkowalleck** [comment]
my issue #62 was deleted, and instead of merging this PR into your working-branch, you closed this one and manually copied it into https://github.com/codacy/codacy-bandit/commit/6a333386c34779d70554bec7b92ed9c097f52bd0

why this strange working model, that does not credit contributors and does not have proper requirements-driven engineering?

**Comment by DMarinhoCodacy** [comment]
Let me apologize for that, it was my fault. I already mentioned in my PR and it's completely fair, since your contribution was 100% right and useful. I had some problems merging your PR, so that's why I copied your changes and created a new one.
Thanks once again and sorry for my mistake.

**Pull request closed without merging**